### DISCUSSION TITLE
[Feature] transfer 4D Softmax layer to onnx

### DISCUSSION THREAD

**Discussion by thinkyang88** [open-mmlab/mmdeploy#1234] [Q&A]
### Motivation

Has anyone transferred 4D softmax layers to onnx? It comes out split into several operators,including reducemax reducesum and so on

**Comment by lvhan028** [comment]
Hi, @thinkyang88 I think it's better to transfer this issue to the [discussion](https://github.com/open-mmlab/mmdeploy/discussions)
Let's find out if there are more users concern.
